lib/os/Kconfig.cbprintf

```diff
@@ -3,6 +3,7 @@
 
 choice CBPRINTF_IMPLEMENTATION
 	prompt "Capabilities of cbprintf implementation"
+	depends on !PICOLIBC
 	default CBPRINTF_COMPLETE
 
 config CBPRINTF_COMPLETE
```

Comment:
This should hold the `depends on !PICOLIBC` and ditto for the `CBPRINTF_NANO`, but cannot place a comment there.